dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/event/EventSearchParams.java

```diff
@@ -90,6 +93,10 @@
     private OrganisationUnit orgUnit;
 
     private OrganisationUnitSelectionMode orgUnitSelectionMode;
+    
+    private AssignedUserSelectionMode assignedUserSelectionMode;
+    
+    private Set<String> assignedUsers;
```

Comment:
If you set this to be an empty set by default, you dont need to do the null check in the other file?

Reply:
The problem is we also have setters which might end up being used as setAssignedUsers(null), and can make it null eventually. Always safe to have a null check.

Reply:
Second thought. Null check for this parameter is done in this file itself. And it might be necessary to check for null, since the setter can be used to set null. I have anyway defaulted that set into an empty set here.